core/capabilities/ccip/ccip_integration_tests/usdcreader/usdcreader_test.go

```diff
@@ -201,7 +207,147 @@ func Test_USDCReader_MessageHashes(t *testing.T) {
 		})
 	}
 }
+func Benchmark_MessageHashes(b *testing.B) {
```

Comment:
Should we add the gobench results above the test as a comment? (as a reference)

Reply:
I agree, benchmark tests provide very few impact if not compared. We could add a tool like benchstat in the CI and fail if we noticed a reg.